docs/en/week03/03-3.md

```diff
@@ -0,0 +1,54 @@
+---
+lang-ref: ch.03-3             
+title: Spiral classification 
+lecturer: Alfredo Canziani 
+authors: Wenhao Li          
+date: 6 May 2021        
+---                    
+
+## [Typora](https://typora.io/)
+Typora is a useful tool to write markdown and latex. It is convenient to write paper and homework, and generating pdf file with Typora.
+
+## [Notion](https://www.notion.so/)
+<center>
+<img src="{{site.baseurl}}/images/week03/03-3/figure1.png" style="background-color:#DCDCDC;" /><br>
```

Comment:
Linked databases is the feature I was trying to stress about. So, perhaps we should use the following instead?
![image](https://user-images.githubusercontent.com/2119355/118657997-f90acf00-b7b9-11eb-88d0-3668d3de967b.png)

I think linking to [this tutorial](https://www.notion.so/Intro-to-databases-fd8cd2d212f74c50954c11086d85997e) would be beneficial. But we need to explain how this should be used to consume online content for keeping up with Deep Learning. I hope you can see my point.

A similar database can be used to keep a reading list for articles one wants to read with its summaries. See work done by [@y0b1byte](https://twitter.com/y0b1byte), namely [this](https://twitter.com/y0b1byte/status/1394394401489707011).